docs/getting_started/explorer.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Explorer
+sidebar_position: 6
+---
+
+From your  Calimero [Console](https://app.calimero.network/dashboard) you can access the explorer which provides you with the ability to view all crypto transactions and information including: accounts, blocks, transactions and receipts, Fungible Tokens and Non-Fungible Tokens in details. 
+
+It is by default a fully private explorer only available to the administrators of the network, but specific information can be shared via API endpoints or shared links through the permissions management service implemented inside the Console.
+
+![](../../static/img/explorer.png)
+
+## Account
+
+Accounts store digital assets, maintain balances, and track transactions.
+
+![](../../static/img/demo_account.png)
+
+By clicking on any of the account ID, you'll be able to have an overview of:
+
+- The number of transactions occured on that account
+- The amount of storage used
+- The native account balance currently on the account
+- The amount of validator stake used
+- The transactio hash details
+- The total amount of fungible tokens the account has
+-  The total amount of non fungible tokens the account has
+
+![](../../static/img/account_details.png)
+
+## Blocks
+
+The blocks serve as a record of transactions and other network activity. Each block in the private shard contains a unique cryptographic hash of the previous block in the chain, which creates a secure and tamper-proof record of all network activity. As new transactions are submitted to the network, they are validated by the network's validators and added to a new block in the chain. Once the block is validated, it is added to the private shard and becomes part of the network's transaction history.
+
+![](../../static/img/blocks.png)
+
+By clicking on any of the blocks, you'll be able to have an overview of:
+
+- The date the transactions occured
+- The account author of transaction
+- The status of the transaction
+- The block and parent hash
+- The transaction amount
+- The receipt of the transaction
+- The gas used for the transaction
```

Comment:
```suggestion
- date the transactions occured
- account author of transaction
- status of the transaction
- block and parent hash
- transaction amount
- receipt of the transaction
- gas used for the transaction
```
lose the `the`.

Reply:
Also, I think plural would fit better with the sentence.
`...an overview of dates the transactions occurred, account authors of transactions, ...`

Reply:
Account author sounds a bit awkward, see if it can be phrased differently